Libraries/Opc.Ua.Client/TraceableSession.cs

```diff
@@ -956,11 +956,11 @@ public ResponseHeader EndAddNodes(IAsyncResult result, out AddNodesResultCollect
         }
 
         /// <inheritdoc/>
-        public Task<AddNodesResponse> AddNodesAsync(RequestHeader requestHeader, AddNodesItemCollection nodesToAdd, CancellationToken ct)
+        public async Task<AddNodesResponse> AddNodesAsync(RequestHeader requestHeader, AddNodesItemCollection nodesToAdd, CancellationToken ct)
```

Comment:
Same as above, it is the caller that needs to await. There are some reasons to use async/await instead of returning a task. See: https://stackoverflow.com/questions/19098143/why-use-async-and-return-await-when-you-can-return-taskt-directly

Reply:
I retract my comment. When using "using", then this is a must.